src/SignalRServiceExtension/Config/SignalRConfigProvider.cs

```diff
@@ -61,11 +55,7 @@ public void Initialize(ExtensionConfigContext context)
                 logger.LogWarning($"Unsupported service transport type: {serviceTransportTypeStr}. Use default {options.AzureSignalRServiceTransportType} instead.");
             }
 
-            serviceManager = new ServiceManagerBuilder().WithOptions(o =>
-            {
-                o.ConnectionString = options.ConnectionString;
-                o.ServiceTransportType = options.AzureSignalRServiceTransportType;
-            }).Build();
+            var serviceManager = CreateServiceManager(options.ConnectionString);
```

Comment:
>CreateServiceManager(options.ConnectionString); [](start = 33, length = 47)

options.ConnectionString could be null and should move 'signalRConnectionInfoAttributeRule.AddValidator(ValidateSignalRConnectionInfoAttributeBinding);' ahead of new serviceManager?

Reply:
If `options.ConnectionString `  == null, then `CreateServiceManager` will return null
We don't want to throw exception in `Initialize`, since connection string may be set in function parameter attribute later